src/topology/discrete_quotient.lean

```diff
@@ -198,6 +200,23 @@ lemma of_le_proj_apply {A B : discrete_quotient X} (h : A ≤ B) (x : X) :
 
 end of_le
 
+/--
+When X is discrete, there is a `semilattice_inf_bot` instance on `discrete_quotient X`
+-/
+instance [discrete_topology X] : semilattice_inf_bot (discrete_quotient X) :=
+{ bot :=
+  { rel := λ a b , a = b,
+    equiv := ⟨λ a, rfl, λ a b h, h.symm, λ a b c h1 h2, h1.trans h2⟩,
+    clopen := λ x, ⟨is_open_discrete _, is_closed_discrete _⟩ },
```

Comment:
It would probably be sensible to add an `is_clopen_discrete` simp lemma to match these other two, making this line
```suggestion
    clopen := λ x, is_clopen_discrete _ },
```

(Yes, I know that saving characters here is arguably a waste of time, but using review as an opportunity to factor out missing lemmas isn't :))

Reply:
Done!